src/org/opendatakit/briefcase/ui/reused/AggregateServerConnectionDialog.java

```diff
@@ -13,13 +14,14 @@
   private final AggregateServerConnectionDialogForm form;
   private final List<Consumer<AggregateServerConnectionConfiguration>> onConnectCallbacks = new ArrayList<>();
 
-  AggregateServerConnectionDialog(AggregateServerConnectionDialogForm form, Predicate<AggregateServerConnectionConfiguration> confValidator, AggregateServerConnectionConfiguration initialConfiguration) {
+  AggregateServerConnectionDialog(AggregateServerConnectionDialogForm form, Predicate<AggregateServerConnectionConfiguration> confValidator, AggregateServerConnectionConfiguration initialConfiguration, BiConsumer<String, String> onSuccess) {
```

Comment:
Why do we need another callback? We already have an `onConnectCallbacks`... Also, why would this inner class execute some code from an ancestor so far apart?